app/Http/Controllers/AdminController.php

```diff
@@ -7,29 +7,108 @@
 use App\Models\User;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Session;
 
 class AdminController extends Controller
 {
     /**
      * Display a listing of the resource.
      */
-    public function index()
-    {
-       return view('Admin.user.admin.allAdmin');
+    // public function index(Request $request)
```

Comment:
Hello Susana, 
Nice to mentor you again:)

I am still not like the unnecessary comments, could you please remove it.